docs/website/docs/general-usage/schema.md

```diff
@@ -126,9 +126,19 @@ coerced in existing column.
 
 ### Data types
 
+| dlt Data Type | Source Value Example |
+| --- | --- |
+| text | `"hello world"` |
+| double | `45.678` |
+| bool | `True` |
+| timestamp | `2023-07-26T14:45:00Z` |
```

Comment:
datetimes may be given via python datetime
- pendulum.now() or datetime.now()
- as ISO strings which get recognized with default schema settings